include/sys/signal.h

```diff
@@ -73,6 +74,13 @@ typedef struct pgrp pgrp_t;
 typedef struct thread thread_t;
 typedef struct ctx ctx_t;
 
+/*! \brief Notify the parent of a change in the child's status.
```

Comment:
We have another function `proc_wakeup_parent` that has very similar description. This one sends a signal and wakes up the parent.